llama_index/llms/konko.py

```diff
@@ -28,15 +29,22 @@
     acompletion_with_retry,
     completion_with_retry,
     from_openai_message_dict,
-    is_chat_model,
-    konko_modelname_to_contextsize,
+    import_konko,
+    is_openai_v1,
     resolve_konko_credentials,
     to_openai_message_dicts,
 )
 from llama_index.llms.llm import LLM
 from llama_index.types import BaseOutputParser, PydanticProgramMode
 
-DEFAULT_KONKO_MODEL = "meta-llama/Llama-2-13b-chat-hf"
+DEFAULT_KONKO_MODEL = "meta-llama/llama-2-13b-chat"
```

Comment:
better to indicate what are the models Konko support?

Reply:
Hi @hatianzhang,

We are constantly adding new models to our inference server, so we implemented a function to gather all the models served by Konko at that particular instance by calling our models endpoint.